apps/stripe/src/app/api/stripe/webhook/route.ts

```diff
@@ -0,0 +1,56 @@
+import { Result } from "neverthrow";
+import { NextRequest } from "next/server";
+
+import { getAndParseStripeSignatureHeader } from "@/app/api/stripe/webhook/stripe-signature-header";
```

Comment:
nitpick: we can have those imports are relative - files are next to route anyway